tools/cross_compile.sh

```diff
@@ -92,6 +92,7 @@ else
           -D CMAKE_BUILD_TYPE="${BUILD_TYPE}" \
           -D REALM_NO_TESTS=1 \
           -D REALM_VERSION="${VERSION}" \
+          -D CPACK_SYSTEM_NAME="${SDK}os" \
```

Comment:
For finer control of the output name it might be worthwhile to use `CPACK_PACKAGE_FILE_NAME="realm-core-${BUILD_TYPE}-${VERSION}-${SDK}os"` instead of changing the system name. That'll produce a file called `realm-core-Debug-4.0.2-iphoneos-devel.tar.gz` So somehow the `-devel` suffix is still getting appended, but the advantage is a bit finer control of the name and using the version passed to the script (I noticed that as it is currently in this PR, the version is produced by git describe, so even if passing an explicit version to the script with `-v 5.1.0-local-test` we'd end up with whatever name git produces in the resulting artefact for example `...-v5.4.0-14-gab1a42e-...`)